metric_providers/lm_sensors/source.c

```diff
@@ -202,7 +203,7 @@ int main(int argc, char *argv[]) {
     setlocale(LC_CTYPE, "");
 
     while (1) {
-        c = getopt_long(argc, argv, "c:f:hts:i:", long_opts, NULL);
+        c = getopt_long(argc, argv, "c:f:hts:i:1", long_opts, NULL);
```

Comment:
I understand the 1 for one record. But this sort of implies that there is also a 2 ... And numbers as args is something I haven't seen that often. Maybe using something like `-n` from head so that you do `-n 1` for record?

Reply:
Jap. I like the `-n 1`.

Will add